extensions/applicationinsights-properties-js/src/PropertiesPlugin.ts

```diff
@@ -56,10 +56,8 @@ export default class PropertiesPlugin extends BaseTelemetryPlugin implements IPr
 
             _initDefaults();
 
-            objDefineProp(_self, "context", {
-                enumerable: true,
-                configurable: true,
-                get: function() {
+            objDefine(_self, "context", {
```

Comment:
Using `objDefine()` as the defaults for `enumerable` and `configurable` are true and it avoids adding these uncompressible names